pkg/config/config_test.go

```diff
@@ -42,7 +42,7 @@ var testDevEnvironment = environment.NewEnvironment("development", "Dev", "", "h
 var testHardeningEnvironment = environment.NewEnvironment("hardening", "Hardening", "", "https://url/to/hardening/environment", "HARDENING")
 var testProductionEnvironment = environment.NewEnvironment("prod-environment", "prod-environment", "production", "https://url/to/production/environment", "PRODUCTION")
 var testManagementZoneApi = api.NewStandardApi("management-zone", "/api/config/v1/managementZones")
-var testHostsAutoUpdateApi = api.NewLegacyApi("hosts-auto-update", "/api/config/v1/hosts/autoupdate")
+var testHostsAutoUpdateApi = api.NewSingleConfigurationApi("hosts-auto-update", "/api/config/v1/hosts/autoupdate")
```

Comment:
I think this can be removed as well, it was only used in `TestIsSkippedDueToConfig`